### PR TITLE
[v1 API Docs] Add page on SPARQL

### PR DESCRIPTION
**Pull request by juliawu**
Adds the missing page on v1/query

Note: This page is almost a 1:1 port of the v0 SPARQL page. If there are wording changes/suggestions, we should consider also updating the v0 page.

### CODE REVIEW

File: api/rest/v1/index.md [hunk 1]
```diff
@@ -88,3 +88,11 @@ Methods for retrieving statistical variable related data.
 | ----------------------------------- | ------------------------------------------------- | --------------------------------------------------- |
 | Variables                           | [/v1/variables](/api/rest/v1/variables)           | Get all variables associated with a specific entity |
 | Variables <bulk-tag>bulk</bulk-tag> | [/v1/bulk/variables](/api/rest/v1/bulk/variables) | Get all variables available for multiple entities   |
+
+### Other Query Languages
```
Comment (shifucun) [inline api/rest/v1/index.md:92]: Maybe " ### Graph Query" and the description below could be specific to SPARQL

File: api/rest/v1/query.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+permalink: /api/rest/v1/query
+---
+
+# /v1/query
+
+Query the Data Commons knowledge graph using
+[SPARQL](https://www.w3.org/TR/rdf-sparql-query/).
+
+This endpoint makes it possible to query the Data Commons knowledge graph using
+SPARQL. SPARQL is a query language developed to retrieve data from websites. It
```
Comment (shifucun) [inline api/rest/v1/query.md:17]: I am not sure if it's for websites, can we borrow something from https://www.w3.org/TR/sparql11-query/

**Review by shifucun — APPROVED** [review]

File: api/rest/v1/index.md (hunk 1 above)
Comment (juliawu) [reply api/rest/v1/index.md]: Changed header to 'Graph Query' and make description specific to SPARQL

File: api/rest/v1/query.md (hunk 2 above)
Comment (juliawu) [reply api/rest/v1/query.md]: Changed the description from 'websites' to 'RDF graph content on the web" for both v1 and v0 pages.

**Pull request merged**